src/main/java/helloworld/HelloWorld.java

```diff
@@ -9,6 +9,6 @@ public static void main(String[] args) {
     }
 
     public static String mavenResponder(){
-        return "Hi, nice to meet you";
+        return "Hi, nice to meet yo";
```

Comment:
I created an issue about this.
```suggestion
        return "Hi, nice to meet yo";
        
        fix this line
```